tests/integration/defs/accuracy/test_disaggregated_serving.py

```
@@ -848,6 +845,66 @@ def test_auto_dtype(self, overlap_scheduler):
             task.evaluate(llm)
 
 
+@skip_pre_hopper
+@pytest.mark.skip_less_device_memory(80000)
+class TestGPTOSS(LlmapiAccuracyTestHarness):
+    extra_evaluator_kwargs = {
+        "fewshot_as_multiturn": True,
+        "apply_chat_template": True,
+    }
```

Ruff (0.13.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)